src/main/java/dukeRobot/Duke.java

```diff
@@ -0,0 +1,104 @@
+import Duke.*;
+import Tasks.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+public class Duke {
+    public static void main(String[] args) {
```

Comment:
Very long function, more than 30 lines of code. Can consider abstracting out each command.

Reply:
Could you please be more specific about the abstracting? Thanks!